notebooks/json_blocks/LUT/BLOCK-T186.ipynb

```diff
@@ -0,0 +1,304 @@
+{
```

Comment:
<p>Can you add a description of what each block is supposed to accomplish or a link to more information on each block?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>